riscv_config/schemas/schema_isa.yaml

```diff
@@ -12907,9 +12907,74 @@ CSR and the value returned in VS-mode or VU-mode.}
               check_with:
               - rv64_check
           default: {accessible: false}
-        
-                  
-    
-          
-          
-                  
+    vxsat:
+      type: dict
+      schema:
+        description:
+          type: string
+          default: The vxsat register records the overflow saturation condition of P and V instructions.
+        address: {type: integer, default: 9, allowed: [9]}
+        priv_mode: {type: string, default: U, allowed: [U]}
+        reset-val:
+          type: integer
+          check_with: max_length
+          default: 0
+        rv32:
+          type: dict
+          schema:
+            fields: {type: list, default: []}
+            ov:
+              type: dict
+              schema:
+                description: {type: string,  default: Stores the state of overflow flag.}
+                shadow: {type: string, default: , nullable: True}
+                shadow_type: {type: string, default: rw, nullable: True, allowed: ['rw','ro']}
+                msb: {type: integer, default: 0, allowed: [0]}
+                lsb: {type: integer, default: 0, allowed: [0]}
+                implemented: {type: boolean, default: true}
+                type:
+                  type: dict
+                  check_with: wr_illegal
+                  oneof:
+                  - schema: { warl: *ref_warl }
+                  - schema: {ro_constant: {type: integer, max: 0 , min : 0}}
+                  - schema: { wlrl: *ref_wlrl }
+                  default:
+                    wlrl:
+                    - 0:1
+              default: {implemented: false}
+              check_with: n_check
```

Comment:
This implies that the `ov` field should only be implemented if the `N` extension is present which doesn't look correct. This should be replaced with a function which checks for the presence of the `P` extension if the field is implemented.

Reply:
Added p_check [4ff617f](https://github.com/riscv-software-src/riscv-config/pull/118/commits/4ff617f428827ff572f085f0524de1999f0d96d2)